constants.py

```diff
@@ -751,3 +751,49 @@
 
 kVelocityControlMotorType = DCMotor.falcon500()
 kVelocityControlkV = 0.01
+
+# constants copied from last year, update when cad is done
+kElevator1CANID = 55
+kElevator1Name = "Elevator1Motor"
+kElevator1PGain = 0.9
+kElevator1IGain = 0
+kElevator1DGain = 0
+kElevator1Inverted = True
+
+kElevatorMaxAccel = 390
+kElevatorMaxVel = 300
+
+kElevator2CANID = 56
+kElevator2Name = "Elevator2Motor"
+kElevator2PGain = 0.12
+kElevator2IGain = 0
+kElevator2DGain = 0
+kElevator2Inverted = False
+
+kElevatorTolerance = 0.05
+
+kMotorPulleyGearRatio = (60 / 18) * (60 / 18)
+
+kPulleyGearPitchDiameter = 0.0572957795
```

Comment:
this number is too precise to be given proper
please make sure you are using the actual pitch diameter of whatever pulley we are using, which is the average between the highest and lowest part of the teeth given as a diameter